### ISSUE TITLE
Missing support for Solana subgraphs

### ISSUE THREAD

**Issue by Chipe1** [graphprotocol/graph-tooling#1578]
### Which packages are impacted by your issue?

@graphprotocol/graph-cli

### Describe the issue

According to this official blog article https://thegraph.com/blog/indexing-solana-substreams/ Solana should be [accessible using the hosted service](https://thegraph.com/blog/indexing-solana-substreams/#how-to-get-started-indexing-solana-data-with-the-graph) but I do not see any option to initialize one using the `graph init` command. I see multiple networks supported for `substreams` protocol but solana is missing.
Is it possible to create a hosted subgraph for Solana using substreams - as mentioned in the blog?

### Reproduction

https://github.com/graphprotocol/graph-tooling

### Steps to Reproduce the Bug or Issue

Initialize a new hosted subgraph with the command `graph init --product hosted-service`.
Select `substreams` as the protocol.
Enter any name.
See the list of "Substreams network" to choose from

### Expected behavior

This list should contain an option for Solana network, but it doesn't

### Screenshots or Videos

This is the list of substream networks I see:
```
? Substreams network … 
❯ mainnet
  goerli
  optimism
  optimism-sepolia
  bsc
  chapel
  poa-core
  gnosis
  fuse
  matic
  fantom
  zksync-era-testnet
  boba
  optimism-goerli
  clover
  moonbeam
  moonriver
  mbase
  fantom-testnet
  arbitrum-one
  arbitrum-goerli
  celo
  fuji
  avalanche
  celo-alfajores
  mumbai
  aurora
  aurora-testnet
  harmony
  base-sepolia
  zksync-era-sepolia
  sepolia
  arbitrum-sepolia
  polygon-zkevm-testnet
  near-mainnet
  near-testnet
  cosmoshub-4
  theta-testnet-001
  osmosis-1
  osmo-test-4
  arweave-mainnet
```

### Platform

- OS: macOS
- NodeJS: v21.6.1
- `@graphprotocol/graph-cli` version: 0.67.2

Output of `graph --version`: `@graphprotocol/graph-cli/0.67.2 darwin-arm64 node-v21.6.1`

**Comment by Tarunshrma** [comment]
@Chipe1 did you get any update on this issue. I still can not see the Solana option in graph cli. I need to work on Solana data indexing using Substream. 

If you figured out I would appreciate if you can provide some pointers for Solana indexing.

Thanks
Tarun Sharma

**Comment by Chipe1** [comment]
@Tarunshrma No, didn't get any update on the issue but was I able to deploy a substream powered subgraph by initializing a mainnet subgraph in the hosted service and then manually editing the network in `subgraph.yaml` to `sol-mainnet`. Here's what my subgraph YAML looks like:
```
specVersion: 0.0.4
description: Solana tracking Subgraph (powered by Substreams)
repository: https://github.com/graphprotocol/graph-tooling
schema:
  file: schema.graphql
dataSources:
  - kind: substreams
    name: solana_substreams_subgraph
    network: sol-mainnet
    source:
      package:
        moduleName: graph_out
        file: solana-substreams-v1.0.1.spkg
    mapping:
      kind: substreams/graph-entities
      apiVersion: 0.0.5
```
I don't remember the exact steps but I think I followed  https://thegraph.com/docs/en/cookbook/substreams-powered-subgraphs/ to initialize my subgraph+substream repo. Hope that helps

**Comment by Tarunshrma** [comment]
thanks @Chipe1 ,

I will try it out.

**Comment by YaroShkvorets** [comment]
Implemented in 0.93 via substreams source
